src/odemis/acq/align/goffset.py

```
@@ -0,0 +1,272 @@
+    """
+    Iteratively adjusts the grating offset to align the spectral peak to the center of the detector.
+    The algorithm estimates the current peak position, calculates the error from the center, and moves the grating offset
+    proportionally to reduce this error.
+    It continues until the peak is within the specified tolerance or the maximum number of iterations is reached.
+    """
+
+    logging.info("Running alignment | detector=%s |",
+                 detector.name)
+
+    try:
+        scale, p0, p1 = estimate_goffset_scale(spgr, detector)
```

Ruff (0.15.4)

[warning] Unpacked variable `p1` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)